src/d3a/models/market/__init__.py

```diff
@@ -39,7 +39,7 @@
 
 log = getLogger(__name__)
 
-TransferFees = namedtuple("TransferFees", ('grid_fee_percentage', 'transfer_fee_const'))
+GridFee = namedtuple("GridFee", ('grid_fee_percentage', 'transfer_fee_const'))
```

Comment:
Sorry, if I missed the discussion, but shouldn't this also the opportunity to rename `transfer_fee_const` here too?
At least internally (we probably need a converter from d3a-web to d3a then)